learning-development/learning-support.qmd

```diff
@@ -10,10 +10,15 @@ As analysts and statistics producers, we require a variety of tools to efficient
 
 For best practice when using software and coding in our process, see our [guidance on RAP expectations](../RAP/rap-expectations.html) and the [DfE Good Code Practice guide](https://dfe-analytical-services.github.io/good-code-practice/index.html).
 
+---
+
 ## Email support
```

Comment:
Is it worth highlighting some of the teams and slack communities here for different tools / groups? Email support is great but isn't the only route, lots of the time others in the community can also help / benefit from seeing it, and also emails aren't the fastest, so teams and IMs can be a lot more effective

Reply:
Are most people allowed to access / use Slack outside of our unit? I don't think many people do

Reply:
Yeah - there's a fully paid for DfE slack and everything. Not many use within our area, but in digital especially it's their main communication route instead of team, but that's why it's good to promote - especially the cross-gov interactions

Reply:
Added some Slack / Teams channels, if you feel like I've missed any then let me know!

Reply:
Think you've hit the main ones!